frontend/src/pages/profile/styles.module.scss

```diff
@@ -0,0 +1,34 @@
+.profile {
+  width: 100%;
+  max-width: 1591px;
+  margin: 0 auto;
+}
+
+.userInfoBlock {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: 270px;
+}
+
+.avatar {
+  width: 270px;
+  height: 269px;
+}
+
+.userInfo {
+  font-size: 32px;
+  line-height: normal;
+}
+
+.userInfo > * {
+  margin: 10px 0;
+}
```

Comment:
If you're overriding default paragraph margin -- then override it globally - in `scaffolding.scss` for * selector declare also `margin: 0` and then apply styling here as it should be intended